app/index.js

```diff
@@ -15,6 +16,8 @@ app.set('view engine', 'jade');
 
 app.locals.title = 'MiniTwit';
 
+app.use(sassCSS('www/stylesheets'));
```

Comment:
The examples on the node-sass-middleware page include more than just a string here

```
var sassMiddleware = require('node-sass-middleware');
...
app.use(sassMiddleware({
    /* Options */
    src: __dirname,
    dest: path.join(__dirname, 'public'),
    debug: true,
    outputStyle: 'compressed',
    prefix:  '/prefix'  // Where prefix is at <link rel="stylesheets" href="prefix/style.css"/>
}));
```